lib/CallBackery/GuiPlugin/Abstract.pm

```diff
@@ -422,7 +423,11 @@ has template => sub {
     monkey_patch $mt->namespace,
         dbLookup => $dbLookup;
     monkey_patch $mt->namespace,
-        slurp => \&slurp;
+        slurp => sub {
+	    my $self     = shift;
```

Comment:
slurp will not be called as a method, so that should be no self

```perl
slurp => sub {
￼ +	    my $filename = shift;
￼ +	    return path($filename)->slurp;
￼ +	},
```